client/html/layouts/panel.html

```diff
@@ -1,15 +1,15 @@
 <template name="PanelLayout">
   {{#if currentUser}}
     <header id="header" class="header">
-      {{> Template.dynamic template=header}}
+      {{> header }}
     </header>
 
     <div id="base">
-      {{> Template.dynamic template=menubar}}
+      {{> menubar }}
       <div id="content">
         {{> Template.dynamic template=main}}
       </div>
-      {{> Template.dynamic template=offcanvas}}
+      {{> offcanvas }}
```

Comment:
When we do:

```html
{{> Template.dynamic template=offcanvas}}
```

We are naming an area offcanvas, and we can put any template inside of this area.

But when we do:

```html
{{> offcanvas }}
```

We are telling to render the template inside on the specified area, no other template can be there besides 'offcanvas'

As there is no reason to use another template in that space besides offcanvas, we changed to this new way, and now we will always render 'ofcanvas' in there.